themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -451,8 +451,11 @@ $d-p-themes: map-merge(
 
 @each $name, $themeSettings in $d-p-themes {
   .theme-#{$name} {
-    > .paragraph,
-    > div[data-geysir-paragraph-id] > .paragraph,
+    > .paragraph:not(.content-moved-inside),
+    > div[data-geysir-paragraph-id] > .paragraph:not(.content-moved-inside),
```

Comment:
tutaj tak naprawdę przywracam to co było wczesniej.. nie wiem w sumie dlaczego to zostało usunięte :thinking: